normandy/selfrepair/static/js/JexlEnv.js

```diff
@@ -0,0 +1,16 @@
+import { Jexl } from 'jexl';
+import { stableSample } from './utils.js';
+
+export default class JexlEnv {
+  constructor(context) {
+    this.context = context;
+    this.jexl = new Jexl();
+    this.jexl.addTransform('date', value => new Date(value));
+    this.jexl.addTransform('stableSample', stableSample);
+  }
+
+  eval(expr) {
+    const oneLineExpr = expr.replace('\n', ' ');
```

Comment:
Probably want to use the slightly-more-complex regex we're applying to filter expressions? Then we can remove it's use there.